src/components/CreateNewProjectWizard/BuildPane.test.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import BuildPane, { BUILD_STEP_KEYS } from './BuildPane';
+import createProject from '../../services/create-project.service';
+
+jest.mock('../../services/create-project.service');
+
+describe('BuildPane component', () => {
+  let wrapper;
+  let instance;
+  const newProject = {
+    projectName: 'New project',
+    projectType: 'create-react-app',
+    projectIcon: 'icon',
+    projectStarter: '',
+  };
+
+  const mockHandleCompleteBuild = jest.fn();
+
+  jest.useFakeTimers();
+
+  const shallowRenderBuildPane = project =>
+    shallow(
+      <BuildPane
+        {...project}
+        status="filling-in-form"
+        handleCompleteBuild={mockHandleCompleteBuild}
+      />
+    );
+
+  const testOutputs = ['Installing packages', 'Dependencies installed'];
+
+  beforeEach(() => {
+    wrapper = shallowRenderBuildPane(newProject);
+    instance = wrapper.instance();
```

Comment:
```suggestion
    instance = wrapper.instance();
    
    // mock console errors so they don't output while running the test
    jest.spyOn(console, 'error')
    console.error.mockImplementation(() => {})
  })

  afterEach(() => {
    console.error.mockRestore()
  })
```

This prevents the error from being output while running the missing project prop test (which is confusing output IMO). From https://github.com/facebook/jest/pull/5267#issuecomment-356605468.

Screenshot of the error being logged:

<img width="300" alt="Screen Shot 2019-03-25 at 7 56 34 PM" src="https://user-images.githubusercontent.com/17421347/54969499-9ac86580-4f3b-11e9-91e4-9aafac6feeb4.png">